src/webui/static/app/dvr.js

```diff
@@ -570,7 +574,9 @@ tvheadend.dvr_finished = function(panel, index) {
                 }
             }],
         tbar: [removeButton, downloadButton, rerecordButton, moveButton],
-        selected: selected
+        selected: selected,
+        grouping: true,
+        groupField: 'disp_title'
```

Comment:
I believe that grouping should be optional (another button?).

Reply:
In case of series recordings, it's better to have it grouped by title, as you can see in this capture, but you can choose make a group from other column and the next time you enter to the tab will continue using the last group used, so I think is not needed create a new button, because the users can configure the grid as they like. 

![finished](https://user-images.githubusercontent.com/9143666/34361311-35abf1fe-ea69-11e7-951f-2fc19279e97e.jpg)

Reply:
Believe me, it won't work on my "production" tvh at all, because the titles are different. Broadcasters puts series and episode info to the title here. Yes, recent EIT scraping might help, but we should expect all situations.